docs/data/toolpad/core/components/page-container/page-container.md

```diff
@@ -0,0 +1,29 @@
+---
+productId: toolpad-core
+title: Page Container
+components: PageContainer, PageContainerToolbar
+---
+
+# Page Container
+
+<p class="description">A component that wraps page container and provides title, breadcrumbs, page actions.</p>
+
+The page container component is the ideal wrapper for your actual dashboard content. It shows the title of the current page and provides breadcrumbs to navigate back into the current hierarchy. It also provides a responsive [container](https://mui.com/material-ui/react-container/) for your page container.
+
+Just like the Dashboard Layout component, the page container component uses the navigation structure that is defined in the app provider to build up its breadcrumbs and title.
+
+{{"demo": "BasicPageContainer.js", "height": 300}}
+
+## Title and Breadcrumbs
+
+The breacrumbs are formed by matching the current pathname with the navigation structure in the App Layout. The title of the matched segment is used as the page title. You can override the page title with the `title` property.
```

Comment:
```suggestion
The breacrumbs are formed by matching the current pathname with the navigation structure defined in the `AppProvider`. The title of the matched segment is used as the page title. You can override the page title with the `title` property.
```

An example of the above will help make it clear; it feels difficult to follow otherwise. For instance, what is the "page title"; what is the "matched segment", and the `title` property on what?

Reply:
Done